programming/motors.md

```diff
@@ -85,63 +90,68 @@ motors at once.
 ~~~~~ python
 # Set one motor to full power in one direction and
 # another to full power in the other:
-R.motor_boards["srABC1"].motors[0].power = 1
-R.motor_boards["srABC1"].motors[1].power = -1
+robot.motor_boards["srABC1"].motors[0].power = 1
+robot.motor_boards["srABC1"].motors[1].power = -1
 
-# Wait a while (perhaps for the robot to move)
+# Wait a while for the robot to move
 time.sleep(3)
 
 # Stop both motors
-R.motor_boards["srABC1"].motors[0].power = 0
-R.motor_boards["srABC1"].motors[1].power = 0
+robot.motor_boards["srABC1"].motors[0].power = 0
+robot.motor_boards["srABC1"].motors[1].power = 0
 ~~~~~
 
 <div class="info" markdown="1">
-  You will need to work out for your robot which values (positive or negative)
-  result in it moving in each direction. This will depend on how you have
-  positioned your motors as well as how they have been wired to the motor board.
+  You will need to work out for your robot which values (positive or negative) result in it moving in each direction.
+  This will depend on how you have positioned your motors as well as how they have been wired to the Motor Board.
```

Comment:
Suggestion: Perhaps a tip here that it's likely easier to swap the wiring than having motors spin in different directions, and that it's easier for humans to have positive values be forward and negative backwards.

Reply:
a40db43